users_ldap_groups/tests/test_users_ldap_groups.py

```diff
@@ -0,0 +1,65 @@
+# -*- coding: utf-8 -*-
+# Copyright 2018 Therp BV <https://therp.nl>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+from mock import Mock, patch
+from odoo.tests.common import TransactionCase
+
+
+@patch('ldap.initialize', return_value=Mock(
+    search_st=Mock(return_value=[
+        ('cn=hello', {'name': ['hello', 'hello2']})
+    ]),
+))
```

Comment:
I'm not sure but this patch call will affect the whole Python runtime, no? Would not be better to use the `start()` method in `setUp()` and `stop()` in `tearDown()`?

Reply:
Oups sorry, didn't see it was a decorator on the test class.

Reply:
thanks for the review, nothing to apologize!